### PR TITLE
Fixed incorrect account object being passed to IconJazz component

### LINKED ISSUE #1760: Account Jazz-icon different then account button
Bildschirmfoto 2021-03-20 um 19 34 05
Bildschirmfoto 2021-03-20 um 19 34 10

### PR DESCRIPTION
**Pull request by gem1no**
fixes #1760 

The issue appears to have been how the `account` object was being generated from `useConnectedWeb3Context()`

it confused the connected wallet's pub key with the xDai proxy account. i replaced the reference to `account` with `context.rawWeb3Context.account` so we get access to the originating address always

### CODE REVIEW

File: app/src/components/modal/modal_your_connection/index.tsx [hunk 1]
```diff
@@ -288,7 +289,7 @@ export const ModalYourConnection = (props: Props) => {
               <TopCardHeaderLeft>
                 <ConnectionIconWrapper>
                   <ConnectorCircle>{connectorIcon}</ConnectorCircle>
-                  <IconJazz account={account || ''} size={28} />
+                  <IconJazz account={context.rawWeb3Context.account || ''} size={28} />
```
Comment (pimato) [inline app/src/components/modal/modal_your_connection/index.tsx:292]: can`t we just use the owner variable which has already be assigned to `context.rawWeb3Context.account` ?

**Review by pimato — CHANGES_REQUESTED** [review]

**Review by KadenZipfel — APPROVED** [review]
Great work!

File: app/src/components/modal/modal_your_connection/index.tsx (hunk 1 above)
Comment (KadenZipfel) [reply app/src/components/modal/modal_your_connection/index.tsx:292]: Looks like the owner variable is used in the function scope, but would be good to assign it at the component level once and use it everywhere that way

**Review by Mi-Lan — APPROVED** [review]
@gem1no Good job it works well! I would also suggest what kaden and geronimo mentioned about including in the component scope and retrieving value from there...

File: app/src/components/modal/modal_your_connection/index.tsx (hunk 1 above)
Comment (gem1no) [reply app/src/components/modal/modal_your_connection/index.tsx:292]: i tried using that var but got error. i will check again

**Review by hexyls — APPROVED** [review]

**Review by pimato — APPROVED** [review]

**Pull request merged**